packages/react/src/generators/setup-ssr/setup-ssr.ts

```diff
@@ -176,7 +176,7 @@ export async function setupSsrGenerator(tree: Tree, options: Schema) {
       },
     },
     serve: {
-      executor: '@nx/webpack:ssr-dev-server',
+      executor: '@nx/react:module-federation-ssr-dev-server',
```

Comment:
I don't think we can do this. This `setup-ssr` is a generator, which may be used in non-module federation apps.